js/index.js

```diff
@@ -20,7 +20,7 @@ INSTRUCTIONS:
     project, when tests are failing, 
     you should get helpful errors 
     along the way!
-    ************/
+    **************/ 
 
 // PLEASE NOTE: Adding global style rules using the * selector, or by adding rules to body {..} or html {..}, or to all elements within body or html, i.e. h1 {..}, has the potential to pollute the test suite's CSS. Try adding: * { color: red }, for a quick example!
```

Comment:
Added proper commits.

Reply:
- [ ] Kindly add a meaningful code and review that; this project aims to learn how to do a review and learn HTML and CSS codes. In giving a code review, please be polite and clear about the inline comment you give feedback on, and explain why it required changing aspects.

Reply:
The problem here is the complete lack of commits, hence the review is to add commits (in future projects)- I believe this was polite and clear, can you clarify what the issue here is?